### PR TITLE
Update actions.ts

### PR DESCRIPTION
**Pull request by senthilsb-toast**
added clear input and uncommented dnd it is working

### CODE REVIEW

File: actions.ts [hunk 1]
```diff
@@ -146,6 +146,10 @@ export async function runSheet(
 
           case "key": await loc.press(d, tos); break;
           case "key:enter": await loc.press("Enter", tos); break;
+          case "key:clear":
```
Comment (svramu) [inline actions.ts:149]: Since we can do this with 'keys' action and empty data, this can be removed.

Comment (svramu) [inline actions.ts:149]: Since we can do this with 'keys' action and empty data, this can be removed.

**Review by svramu — CHANGES_REQUESTED** [review]
trying this process

**Comment by senthilsb-toast** [comment]
Not Required

**Pull request closed without merging**